src/blocks/ImageBlock/Image.astro

```diff
@@ -27,39 +27,42 @@ const { image, loading = 'lazy' } = Astro.props;
 const altText = image.alt ? image.alt : ''; // default to empty string for accessibility
 const imageUnavailableMessage = t('image_unavailable');
 ---
+<enhanced-image>
```

Comment:
Since the `Image` is part of the `ImageBlock` component, I would name the custom element `image-block`. That's consistent with other custom elements in the repository and predictable when someone encounters the custom element in the code that's shipped (like in the browser). I understand it's a bit different than with other blocks as it's not the top level element in `ImageBlock.astro`, but I'm okay with that :)